presto-raptor/src/main/java/com/facebook/presto/raptor/backup/BackupManager.java

```diff
@@ -84,10 +114,24 @@ public void shutdown()
             return completedFuture(null);
         }
 
+        // Increment the notifying counter before making the async call
         // TODO: decrement when the running task is finished (not immediately on cancel)
```

Comment:
@electrum 
Can we remove this TODO since we do want to decrease the counter right after the future is canceled?